packages/lesswrong/components/posts/EAPostsItem.tsx

```diff
@@ -22,12 +22,14 @@ import PostReadCheckbox from "./PostReadCheckbox";
 import PostsItemNewCommentsWrapper from "./PostsItemNewCommentsWrapper";
 import PostMostValuableCheckbox from "./PostMostValuableCheckbox";
 import { maybeDate } from "@/lib/utils/dateUtils";
+import { defineStyles } from "../hooks/defineStyles";
+import { useStyles } from "../hooks/useStyles";
 
 const KARMA_WIDTH = 50;
 const CARD_IMG_HEIGHT = 80;
 const CARD_IMG_WIDTH = 160;
 
-export const styles = (theme: ThemeType) => ({
+export const styles = defineStyles("EAPostsItem", (theme: ThemeType) => ({
```

Comment:
I think these need to be separated into a separate export, otherwise the style priority will get used by whatever's importing it (which I assume wasn't the case before)